nncf/experimental/tensor/__init__.py

```diff
@@ -12,5 +12,6 @@
 from nncf.experimental.tensor.enums import TensorBackendType
 from nncf.experimental.tensor.enums import TensorDataType
 from nncf.experimental.tensor.enums import TensorDeviceType
+from nncf.experimental.tensor.functions import *  # pylint: disable=redefined-builtin
```

Comment:
Could you briefly explain why you need this import?

Reply:
removed, import will be discussed in next PR when the Tensor module will be retired from experimental status